From 02fc13fb6611b68d7d3275777f5674192b2f2bb5 Mon Sep 17 00:00:00 2001
From: Phillip Susi <psusi@ubuntu.com>
Date: Sun, 17 Nov 2013 22:45:39 -0500
Subject: [PATCH 63/89] Fix test compilation

The tests were not being linked against libpthread but were using
it, and recent versions of gcc refuse to implicitly pull it in.
---
 libparted/tests/Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/libparted/tests/Makefile.am b/libparted/tests/Makefile.am
index bfa5790..12abe60 100644
--- a/libparted/tests/Makefile.am
+++ b/libparted/tests/Makefile.am
@@ -10,7 +10,8 @@ AM_CFLAGS = $(WARN_CFLAGS) $(WERROR_CFLAGS)
 
 LDADD = \
   $(top_builddir)/libparted/libparted.la \
-  $(CHECK_LIBS)
+  $(CHECK_LIBS) \
+  -lpthread
 
 AM_CPPFLAGS = \
   $(CHECK_CFLAGS) \
-- 
1.8.5.3

